pkg/agent/flowexporter/connections/baseconnections.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connections
+
+import (
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
+	"github.com/vmware-tanzu/antrea/pkg/agent/proxy"
+)
+
+type BaseConnectionStore interface {
```

Comment:
Do we need the other separate interfaces for ConnectionStore and DenyConnectionStore?
Can we have one interface and three different structs implementing the single interface (BaseConnectionStore, ConnectionStore and DenyConnectionStore)? We can have separate methods for each struct that are not part of interface.
Interface for ConnectionStore is just needed for creating a mock in exporter_test.go

Reply:
Or just create the interface for ConnectionStore which is required for mocks and the rest you can keep as just struct objects.

Reply:
Sure. Please refer to the latest commit for the change. Thanks!